debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/naming/TableNamingStrategy.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc.naming;
+
+import java.util.Map;
+
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import io.debezium.bindings.kafka.KafkaDebeziumSinkRecord;
+import io.debezium.connector.jdbc.JdbcSinkConnectorConfig;
+import io.debezium.sink.SinkConnectorConfig;
+import io.debezium.sink.naming.CollectionNamingStrategy;
+
+/**
+ * A pluggable strategy contract for defining how table names are resolved from kafka records.
+ *
+ * @author Chris Cranford
+ */
+public interface TableNamingStrategy extends CollectionNamingStrategy {
```

Comment:
@jpechane @Naros I have added another commit to re-add the old API, mapping to the new implementation. Wdyt?